README.md

```diff
@@ -21,49 +43,138 @@ Install via npm:
+| rtl         | _boolean_                                                                                | false                           | Right-to-left option for rtl languages, e.g. Arabic.                                                                                                                                                                              |
+| orientation | _{axis:<br>"both"\|"bottom" \| "none" \| "top",<br>item: "top" \| "bottom"}_             | {axis:"bottom", item: "bottom"} | There are a number of orientation options for the time axis and the items.                                                                                                                                                        | [Example](https://github.com/visjs/vis-timeline/blob/master/examples/timeline/styling/axisOrientation.html) \| [Demo](https://visjs.github.io/vis-timeline/examples/timeline/styling/axisOrientation.html) |
+| **EDITING** |                                                                                          |                                 |                                                                                                                                                                                                                                   |
+| editable    | _boolean \| {add: boolean, updateTime: boolean, updateGroup: boolean, remove: boolean }_ | false                           | An editable timeline allows to drag items around, create new items, and remove items. Changes are logged in the browser console.                                                                                                  |                                                                                                                                                                                                            |
+| multiselect | _boolean_                                                                                | false                           | **Editable** option is required.<br><br>Allow selecting multiple items using ctrl+click, shift+click, or hold.                                                                                                                    |
+| snap        | _function(date: Date, scale: string, step: number) \| null_                              | null                            | **Editable** option is required.<br><br>Snap function controls timeline item alignment:<br>- Returns modified Date object<br>- Called when items are moved<br>- Affects time unit snapping<br>- Set to _null_ to disable snapping | See [Snap Example](#snap-example) for implementation                                                                                                                                                       |
```

Comment:
The options are already described in [the docs](https://github.com/visjs/vis-timeline/blob/master/docs/timeline/index.html) and should not be part of the readme. Otherwise we have two places we need to maintain them.